project/src/main/java/com/google/sps/data/PlacesScorer.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.maps.model.LatLng;
+
+public interface PlacesScorer {
+
+    ImmutableMap<Place, Double> getScores(ImmutableList<Place> places, LatLng userLocation);
```

Comment:
Added the javadoc. Since there is no extra information in the javadoc of this function on the Impl class I removed it from there (I understand it inherits this javadoc automaticaly). Is that right?

Reply:
Correct!